### PR TITLE
Mpotane patch 1

### PR DESCRIPTION
**Pull request by mpotane**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to me! 👍

**Review by iamthecloverly — APPROVED** [review]

**Review by stevenovak123 — APPROVED** [review]

File: data/mpotane.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     },
```
Comment (priyankarpal) [inline data/mpotane.json:18]: I think you need to delete these two brackets & make it clear. It will help to read.

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/mpotane.json (hunk 1 above)
Comment (mpotane) [reply data/mpotane.json:18]: Which one bro? I don't see brackets in the suggested changes to be made.

Comment (priyankarpal) [reply data/mpotane.json:18]: Screenshot from 2023-02-03 22-37-57

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/mpotane.json (hunk 1 above)
Comment (mpotane) [reply data/mpotane.json:18]: I think it doesn't show the lines 14 to 17. but it has properties in it.

Comment (priyankarpal) [reply data/mpotane.json:18]: ok

**Review by Panquesito7 — APPROVED** [review]
Nice! Thanks. 🎉

**Pull request merged**